### PR TITLE
Hide '0 modules with unused exports' message

### PR DESCRIPTION
**Pull request by thewilkybarkid**
Currently a successful run outputs a '0 modules with unused exports'; similar tools like ESLint, Stylelint and TypeScript don't output anything if this happens.

### CODE REVIEW

**Comment by mrseanryan** [comment]
@pzavolinsky should this be an option do you think?

Some users may already rely on this "0 issues output"

**Comment by pzavolinsky** [comment]
Agreed, something like `silent` or `quiet`

**Comment by thewilkybarkid** [comment]
Happy to add an option, but if #161 triggers a new major version might not be needed?

**Comment by mrseanryan** [comment]
@thewilkybarkid can you please add an option `--silent` - for an example see `exitWithCount`.

Then need to mention that in the README, and also in the CHANGELOG.

thanks!
Sean

**Comment by thewilkybarkid** [comment]
@mrseanryan Hopefully all sorted now!

File: src/cli.ts [hunk 1]
```diff
@@ -31,6 +31,12 @@ export const runCli = (
 
     const files = Object.keys(analysis);
 
+    const options = extractOptionsFromFiles(tsFiles).options;
+
+    if (options?.silent && files.length === 0) {
+      return exitWith(ExitCode.NoUnusedExportsFound);
```
Comment (mrseanryan) [inline src/cli.ts:37]: bug - exiting here will break `exitWithCount`

Also - unfortunately an ongoing issue with travis means this PR is kind of stuck.

I will make a new PR that cherry picks the commits ...

**Comment by mrseanryan** [comment]
closing - replaced with #168 

thanks for contributing!

**Pull request closed without merging**